params/config.go

```diff
@@ -1168,6 +1169,7 @@ func init() {
 	testnetBytePrefixList[12301] = []int{100, 109}
 	testnetBytePrefixList[12302] = []int{110, 119}
 	testnetBytePrefixList[12303] = []int{120, 129}
+	testnetBytePrefixList[1337] = []int{0, 129} // for integration tests
```

Comment:
What is the purpose of the new 1337 chain? Why not use one of the existing testnet chains?

Reply:
Decided to go with using the Mainnet chain IDs.